### PR TITLE
Mtg DecisionTree fixes, updating and WIP Votings

### PR DESCRIPTION
**Pull request by tkenakka**

### CODE REVIEW

**Comment by tkenakka** [comment]
Deployed to dev server.

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 1]
```diff
@@ -98,12 +98,12 @@ angular.module('dashboard')
             $log.debug("meetingStatusCtrl: canMeetingToBeClosed");
             var result = true;
             if (angular.isObject(self.mtgDetails) && angular.isArray(self.mtgDetails.topicList)) {
-                self.mtgDetails.topicList.forEach(function (t) {
-                    if (angular.isObject(t) && t.topicStatus !== CONST.TOPICSTATUS.READY.stateId) {
+                for (var i = 0; result && i < self.mtgDetails.topicList.length; i++) {
```
Comment (evidentti) [inline app/scripts/controllers/meeting.status.ctrl.js:101]: ?

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 2]
```diff
@@ -474,8 +482,10 @@ angular.module('dashboard')
                         }
 
                     }, function (error) {
-                        $log.error("meetingStatusCtrl.setMeetingStatus: " + JSON.stringify(error));
-                        // todo: implement error handling
+                        $log.error("meetingStatusCtrl.setMeetingStatus: ", arguments);
+                        if (angular.isObject(error)) {
+                            Utils.showErrorForNum(error.errorCode);
+                        }
```
Comment (evidentti) [inline app/scripts/controllers/meeting.status.ctrl.js:488]: Some default error note if 'error' is not an object?

File: app/scripts/decisions.directive.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ angular.module('dashboard')
                         self.voting = resp.voting;
                     }, function (error) {
                         $log.error("dbDecisions.getDecisions ", arguments);
-                        self.error = error.error;
+                        self.error = error.errorCode;
```
Comment (evidentti) [inline app/scripts/decisions.directive.js]: self.error => self.errorCode ?

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 4]
```diff
@@ -528,10 +538,9 @@ angular.module('dashboard')
                                 }, this);
                             }
                         }, function (error) {
-                            $log.error("meetingStatusCtrl.setTopicStatus: " + JSON.stringify(error));
+                            $log.error("meetingStatusCtrl.setTopicStatus: ", arguments);
                             if (angular.isObject(error)) {
-                                var str = Utils.stringIdForError(error.error);
-                                DialogUtils.showError("STR_ERR_TITLE", str);
+                                Utils.showErrorForNum(error.errorCode);
```
Comment (evidentti) [inline app/scripts/controllers/meeting.status.ctrl.js]: showErrorForNum => showErrorForErrorCode ?

**Comment by evidentti** [comment]
LGTM with comments

**Comment by tkenakka** [comment]
Thanks!
Comments fixed, rebased, PR updated. Merging.

**Pull request merged**